### PR TITLE
print gcc version during install, towards #3291

### PR DESCRIPTION
**Pull request by jangorecki**
towards #3291

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -77,5 +77,10 @@ else
   sed -e "s|@openmp_cflags@|\$(SHLIB_OPENMP_CFLAGS)|" src/Makevars.in > src/Makevars
 fi
 
+# compiler info to output #3291
+if [ "$CC"=~"gcc" ]; then
+  GCCV=`${CC} -dumpfullversion -dumpversion`
+  echo "$CC $GCCV"
+fi
```
Comment (jangorecki) [inline configure:84]: @MichaelChirico any idea if it works in macosx bash?
there is a report https://github.com/Rdatatable/data.table/issues/4410#issuecomment-627073357
which seems to not print gcc version (see in link). I double checked that `/usr/local/gfortran/bin/gcc` satisfies `"$CC"=~"gcc"` condition on my ubuntu.

Comment (MichaelChirico) [reply configure:84]: What am I looking for? here is my output on OSX before the `clang ... -c -o` commands

```
R CMD INSTALL . --preclean
* installing to library ‘/Library/Frameworks/R.framework/Versions/4.0/Resources/library’
* installing *source* package ‘data.table’ ...
** using staged installation
zlib 1.2.11 is available ok
OpenMP supported
/usr/local/opt/llvm/bin/clang -fopenmp 4.2.1
** libs
```

I guess that second-last line is the `gcc` printout

Comment (jangorecki) [reply configure:84]: we don't detect clang, user is using gcc but for some reason it is still not being printed to output

Comment (MichaelChirico) [reply configure:84]: I also tried using `/usr/local/gfortran/bin/gcc` instead, here's my `~/.R/Makevars` after:

```
# The following is added per data.table/wiki/Installation
#LLVM_LOC = /usr/local/opt/llvm
GCC_LOC=/usr/local/gfortran
#CC=$(LLVM_LOC)/bin/clang -fopenmp
#CXX=$(LLVM_LOC)/bin/clang++ -fopenmp
CC=$(GCC_LOC)/bin/gcc -fopenmp
CXX=$(GCC_LOC)/bin/g++ -fopenmp
CFLAGS=-g -O3 -Wall -pedantic -std=gnu99 -mtune=native -pipe
CXXFLAGS=-g -O3 -Wall -pedantic -std=c++11 -mtune=native -pipe
#LDFLAGS=-L/usr/local/opt/gettext/lib -L$(LLVM_LOC)/lib -Wl,-rpath,$(LLVM_LOC)/lib
#CPPFLAGS=-I/usr/local/opt/gettext/include -I$(LLVM_LOC)/include
LDFLAGS=-L/usr/local/opt/gettext/lib -L$(GCC_LOC)/lib -Wl,-rpath,$(GCC_LOC)/lib
CPPFLAGS=-I/usr/local/opt/gettext/include -I$(GCC_LOC)/include
```

and then the same preamble output from `R CMD INSTALL . --preclean`:

```
* installing to library ‘/Library/Frameworks/R.framework/Versions/4.0/Resources/library’
* installing *source* package ‘data.table’ ...
** using staged installation
zlib 1.2.11 is available ok
OpenMP supported
/usr/local/gfortran/bin/gcc -fopenmp 6.3.0
** libs
```

Comment (jangorecki) [reply configure:84]: OK, it is casued by early exit
```
if [ $msg -ne 0 ]; then
  ...
  exit 0
```

**Pull request merged**